client-data/tools/clear/clear.js

```diff
@@ -43,6 +43,7 @@
                 Tools.drawingArea.innerHTML = '';
                 break;
         }
+        updateWhiteboardBackground();
```

Comment:
Does this add the whiteboard background image into the projected image from the robot projector? We don't want that.